RELEASE_STEPS.md

```diff
@@ -0,0 +1,54 @@
+# Release Notes publishing checklist
+
+- Add the release note text inside `_includes/releases` folder as a markdown file, without headers.
+  As the contents of these files are raw, they can be checkout from external sources (eg, once the docs have 
+  been move to the extensions repository).
+
+- Add the version details in `_versions`, with the following content and updating the corresponding
+  version number and the correct filename in the `explanation` attribute (without the `_includes` parent
+  directory):
+
+```markdown
+---
+date: 2022-01-01
+
+title: Babelfish 0.1.0
+product: Babelfish for PostgreSQL
+version: '0.1.0'
+
+components:
+  -
+    role: engine
+    artifact: babelfish
+    version: 0.1.0
+sections:
+  source:
+    explanation: "/releases/babelfish-0.1.0.md"
+pretty:
+  artifacts:
+    babelfish: ''
+  sections:
+    source: 'Babelfish 0.1.0'
+---
+Babelfish for PostgreSQL is open source software that uses the Apache License version 2 (ALv2) and PostgreSQL Licence. ALv2 grants you well-understood usage rights; you can use, modify, extend, embed, monetize, resell, and offer Babelfish for PostgreSQL as part of your products and services. The source for the entire project is available on [GitHub](https://github.com/babelfish-for-postgresql) and you're welcome to build from source for customized deployments. 
```

Comment:
Do we really want this much text for a mouse-over? It seems excessive. Any thoughts @susanmdouglas-aws

Reply:
Way long - how about:

Babelfish for PostgreSQL is open source software that uses the Apache License version 2 (ALv2) and PostgreSQL License. Project source is freely available on [GitHub](https://github.com/babelfish-for-postgresql).